Pocket/pocket-ios
# SwiftUI Home - Scrolling improvements

## Goal
This PR adds some refactors to improve scrolling on SwiftUI Home
* Remove the `Item` query from CardView, since it carries only info that won't need to be observed for changes
* Remove the end of feed animation for now

## Test Steps
* Build/run
* Go to SwiftUI Home
* Make sure the scrolling is smooth
